tests/countries/test_pakistan.py

```diff
@@ -203,3 +203,43 @@ def test_2022(self):
             ("2022-11-09", "Iqbal Day"),
             ("2022-12-25", "Quaid-e-Azam Day"),
         )
+
+    def test_l10n_ur_pk(self):
+        self.assertLocalizedHolidays(
+            "ur_PK",
+            ("2022-02-05", "یوم یکجہتی کشمیر"),
+            ("2022-03-23", "یوم پاکستان"),
+            ("2022-05-01", "یوم مزدور"),
+            ("2022-05-03", "عید الفطر"),
+            ("2022-05-04", "عید الفطر"),
+            ("2022-05-05", "عید الفطر"),
+            ("2022-07-10", "عید الاضحی"),
+            ("2022-07-11", "عید الاضحی"),
+            ("2022-07-12", "عید الاضحی"),
+            ("2022-08-08", "عاشورہ"),
+            ("2022-08-09", "عاشورہ"),
+            ("2022-08-14", "یوم آزادی"),
+            ("2022-10-09", "عید میلاد النبی"),
+            ("2022-11-09", "یوم اقبال"),
+            ("2022-12-25", "یوم قائداعظم"),
+        )
+
+    def test_l10n_en_us(self):
```

Comment:
These methods should be placed in alphabetical order. Look at other localized countries tests.